spec/requests/listings_spec.rb

```diff
@@ -537,25 +537,6 @@
     }
   end
 
-  describe "GET /listings" do
```

Comment:
These tests are no longer valid as the DOM node referenced by the id `"listing-filters"` is now rendered in a Preact component.

Reply:
I'll pop up a subsequent PR to address this as it also requires a refactoring of `<Listings />`.